<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="OverviewWidget" name="widget_3" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChartWidget" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="apiKey">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="secKey">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button">
          <property name="text">
           <string>매매시작</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button2">
          <property name="text">
           <string>매매시작</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button3">
          <property name="text">
           <string>매매시작</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tickerKey">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button4">
          <property name="text">
           <string>코인변경</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_21" native="true">
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border : 1px solid white;
border-radius : 0px;
background-color:  white;
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item>
            <widget class="MybalancesWidget" name="widget_4" native="true">
             <widget class="QLabel" name="label_15">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>71</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>13</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>내 보유코인</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="OrderbookWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrderbookWidget</class>
   <extends>QWidget</extends>
   <header>orderbook</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartWidget</class>
   <extends>QWidget</extends>
   <header>chart</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OverviewWidget</class>
   <extends>QWidget</extends>
   <header>overview</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MybalancesWidget</class>
   <extends>QWidget</extends>
   <header>mybalances</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
